.cirrus.yml

```diff
@@ -153,6 +165,8 @@ task:
         - name: netbsd-postgres
           depends_on:
             - netbsd-vanilla
+          on_failure:
+            <<: *on_failure_googlecompute
 
```

Comment:
Why don't we just do this at the task: level, instead of repeating it in multiple matrices?